pkgs/account/src/lib/ui/onboarding-wallet-info.svelte

```diff
@@ -42,48 +41,47 @@
 </script>
 
 <div class="wallet-info-container">
-  <span class="connected">Connected</span>
+  <Card>
+  <div class="wallet-header">
+    <span class="connected">Connected</span>
+    <span class="status-dot" aria-hidden="true"></span>
+  </div>
   <div class="wallets-stack">
     {#if accountDataProvider.isConnected}
-      <Card>
-        <div class="account-details">
-          <div class="wallet-label">Starknet</div>
-          <p>{shortenHex(padAddress(accountDataProvider?.address ?? ''), 4)}</p>
-          <button
-            onclick={() => {
-              accountManager?.disconnect();
-            }}
-            aria-label="Logout Starknet"
-          >
-            <img src={Icons.logout.default} alt="logout" class="logout-icon" />
-          </button>
-        </div>
-      </Card>
+      <div class="account-details">
+        <div class="wallet-label">Starknet</div>
+        <p>{shortenHex(padAddress(accountDataProvider?.address ?? ''), 4)}</p>
+        <Button
+          onclick={() => accountManager?.disconnect()}
+          aria-label="Logout Phantom"
```

Comment:
```suggestion
          aria-label="Logout Starknet"
```

It's next to the starknet button